### ISSUE TITLE
GeophysicalModel

### ISSUE THREAD

**Issue by mbeaufils** [opengeospatial/Geotech#31]
This model would be composed of several #30

**Comment by Didymograptus** [comment]
I think that we need consider the use of a 'geophysical model' quite carefully. Why would we have a geophysical model and not an insitu-testing model, or a lab-testing model? Geophysics is just a method of exploration, used to inform the creation of the geological model or geotechnical model by filling in the gaps between known data points. It has inherent uncertainty built in to it, that the experienced engineer or geologist analyses to form knowledge.

**Comment by mbeaufils** [comment]
> I think that we need consider the use of a 'geophysical model' quite carefully. Why would we have a geophysical model and not an insitu-testing model, or a lab-testing model? Geophysics is just a method of exploration, used to inform the creation of the geological model or geotechnical model by filling in the gaps between known data points. It has inherent uncertainty built in to it, that the experienced engineer or geologist analyses to form knowledge.

@Didymograptus 
I agree that geophysics is at first measurements and so belong to Book A. As answered to Hiro last week, the proposal is to have geophysical measurements connected to either an Observed Zone or a Borehole / trial pit.

This geophysical model discussion as part of Book B, is about the wish to declare some interpreted model that will contain units based on geophysical properties (not necessarily geological, hydrogeological or geotechnical as the other units).
